### PR TITLE
Adds tethys_dask_scheduler

### PR DESCRIPTION
**Pull request by swainn**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by swainn** [comment]
@conda-forge/help-python this recipe should be good to go!

File: recipes/tethys_dask_scheduler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - click
+    - distributed
```
Comment (xhochy) [inline recipes/tethys_dask_scheduler/meta.yaml:22]: These are probably only needed in `run`
```suggestion
```

File: recipes/tethys_dask_scheduler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+test:
+  imports:
+    - tethys_dask_scheduler
+    - tethys_dask_scheduler.plugin
+  requires:
+    - pip
+    - click
+    - distributed
+  commands:
```
Comment (xhochy) [inline recipes/tethys_dask_scheduler/meta.yaml:36]:
```suggestion
  commands:
```

**Comment by swainn** [comment]
@xhochy done!

**Pull request merged**